src/data/pokemon-evolutions.ts

```diff
@@ -88,21 +88,88 @@ export class SpeciesEvolutionCondition {
   }
 }
 
-// TODO: Break this up into a MaleCondition and FemaleCondition?
-export class GenderEvolutionCondition extends SpeciesEvolutionCondition {
-  constructor(requiredGender: Gender) {
-    super((p) => p.gender === requiredGender);
-    this.description = "requires gender";
+/**
+ * For Pokemon that require being male, including
+ * Gallade
+ * Mothim
+ * Basculegion
+ * Oinkalogne
+ * Meowstic
+ *
+ * Custom:
+ * Glalie
+ * Huntail
+ */
+export class MaleEvolutionCondition extends SpeciesEvolutionCondition {
+  constructor() {
+    super((p) => p.gender === Gender.MALE);
+    this.description = "requires male";
   }
 }
 
+/**
+ * For Pokemon that require being female, including
+ * Froslass
+ * Wormadam, Vespiquen
+ * Basculegion
+ * Salazzle
+ * Oinkalogne
+ * Meowstic
+ *
+ * Custom:
+ * Gardevoir
+ * Gorebyss
+ */
+export class FemaleEvolutionCondition extends SpeciesEvolutionCondition {
+  constructor() {
+    super((p) => p.gender === Gender.FEMALE);
+    this.description = "requires female";
+  }
+}
+
+/**
+ * For Pokemon that have mainline evolutions requiring day time. Includes
+ * Eevee
+ * Budew, Happiny, Riolu
+ * Tyrunt
+ * Yungoos
+ * Rockruff, Formantis
+ * Hisui Sneasel
+ *
+ * Also includes Pokemon that have alternate forms like
+ * Cubone, Koffing, Mime Jr
+ * Quilava
+ * Wurmple
+ * Samurott, Rufflet
+ * Goomy
+ * Dartrix, Cosmoem
```

Comment:
```suggestion
 * For Pokemon that have mainline evolutions requiring day time. Includes:
 * ```
 * Eevee
 * Budew, Happiny, Riolu
 * Tyrunt
 * Yungoos
 * Rockruff, Formantis
 * Hisui Sneasel
 * ```
 *
 * Also includes Pokemon that have alternate forms like:
 * ```
 * Cubone, Koffing, Mime Jr
 * Quilava
 * Wurmple
 * Samurott, Rufflet
 * Goomy
 * Dartrix, Cosmoem
 * ```
```